src/check_hooks.c

```diff
@@ -55,7 +55,7 @@ enum selint_error add_check(enum node_flavor check_flavor, struct checks *ck,
 	return SELINT_SUCCESS;
 }
 
-enum selint_error call_checks(struct checks *ck,
+enum selint_error call_checks(const struct checks *ck,
```

Comment:
Do we want this to be marked `const`?
We pass the member `check_nodes` to `call_checks_for_node_type()`, which modifies the member